### DISCUSSION TITLE
Centralising notes from the accessibility working group

### DISCUSSION THREAD

**Discussion by malvikasharan** [the-turing-way/accessibility-working-group#1] [General]
### Summary

# Accessibility Working Group

This discussion topic centralises notes for the accessibility working group: https://pad.sfconservancy.org/p/ttw-accessibility-wg

**Comment by malvikasharan** [comment]
Draft of 'accessibility for events' is shared in Etherpad for direct edit: https://pad.sfconservancy.org/p/ttw-etherpad

(@aleesteele is editing this to keep track of what is in this pad - preliminary meetings): https://pad.sfconservancy.org/p/ttw-accessibility-wg)

# 27-04-2023
Attendees: Andrea Sanchez-Tapia (@AndreaSanchezTapia), Liz Hare (@LizHareDogs), Alexandra Araujo Alvarez (@AlexandraAAJ)

Notes from ALS (who could not attend!)
- Excited for folks to meet Ale - use this time for introductions and context if helpful
- Considering a themed month of campaigning around accessibility this year - during the summer or autumn? Themed work during Collaboration Cafes or otherwise
- Starting an ACCESSIBILITY.MD file on Github - documentation for accessibility practices, and how we want to engage as a community
- Draw from slack policies: https://hackmd.io/@turingway/ttw-slack-intro

Campaigning for Accessibility: Instead of celebrate, show the right practices. Setting up good practices in real life

Liz 
- People interested in the topic
- Accessibility for people with disabilities: we do to make everyones opportunities equal and the idea of celebrating can be touch. It should be happening 
- Github will set up an improve accessibility. There's a position open for that.  
    
Andrea: Use the book dash to prove that we have 
- She will check if the Alt text from the Book Dash  was correct, the activity to improving the Alt Text, how to do better Alt text. 

Bookdash:
- she would like to see that next bookdash has normalised practices. 
- localisation and translation: making it part of the practice
- try to make the bookdash in the most accessible way
- guidelines about Slack usage
- links need to be at the top of messages
-  the activity to improving the Alt Text, how to do better Alt text. Avoid cultural language
- Bookdash recording and transcript and captions, we should leave them on.
- Data visualisation online: when people share their screen, that is not accessible. So slides should be shared in advance. 

Next Steps:
- Andrea will be able to help, plan for the pre book Dash Coordination. She will be in London from the 15 to 20 May
- Is Zenodo good for accessibility? / Liz will check 
- Jonny writes words: make sure that he uses the accessibility format
- Scriberia: live-audio description was done by Andrea - Ale, will raise this 

Resources:
- Accessibility tag on Github: https://github.com/alan-turing-institute/the-turing-way/issues?q=is%3Aopen+is%3Aissue+label%3Aaccessibility
- Notes from November 2022 Book Dash: https://pad.sfconservancy.org/p/ttw-accessibility-notes
- Presentation from Book Dash
- Youtube recording: https://www.youtube.com/watch?v=ARMs7C_wE04&t=1s
- Slides from LH: https://lizharedogs.github.io/ttwBookDashNov22accessibility/#1
- Slides from AST: https://andreasancheztapia.github.io/BookDash2022/#1
- Starting new book around accessibility: https://github.com/alan-turing-institute/the-turing-way/issues/2730
- Slack policies: https://hackmd.io/@turingway/ttw-slack-intro

# 30-05-2023
**Attendees**: Liz Hare (@LizHareDogs), Andrea Sanchez-Tapic (@AndreaSanchezTapia), Liz Hare (@LizHareDogs), Alexandra Araujo Alvarez (@AlexandraAAJ), Laurel Ascenzi, Anne Lee Steele (@aleesteele)

Discussions
- Updates about the Book dash
- Set up a day or 2 a month to meet
-  Once a month, the 3rd Thursday 17:00 UK time, Ale will liase with Liz, Anne, Laurel and Andrea to find the right time. Decision will be made asynchronously (with Metadocencia teams)
- 1 strategy meeting a month, 1 coworking? - 2nd wednesday or Thursday (?)
- Discussions around how we do the WG: https://github.com/alan-turing-institute/the-turing-way/discussions/3076
- Proposed guide for accessibility: https://github.com/alan-turing-institute/the-turing-way/issues/2730
- Need to slow down existing work to ensure we can move more carefully & mindfully
- Metadocencia: experts from team invited with lived experiences
- Accessibility.md: proposing our own proposals as a community - https://github.com/alan-turing-institute/the-turing-way/issues/3145
- Book Dash Accessibility - improvement from 2022?
- Feedback from Book Dash? https://pad.sfconservancy.org/p/ttw-etherpad
- Funding / Resources
- Consultancy + Funding for Liz? Can we get that?
- Liz Jackson has a consulting company - https://www.core77.com/posts/89571/Liz-Jackson-Doesnt-Want-Design-for-Disability%E2%80%94She-Wants-Disabled-People-to-Be-Part-of-Designing-Better-Products
- Foundations of WG
- Andrea & Liz were a part of the useR! Conference, and wrote this paper about their practices: https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010164

Notes from previous discussions:
- https://pad.sfconservancy.org/p/ttw-accessibility-notes
- UseR! Knowledge base: https://rconf.gitlab.io/userknowledgebase/main/
- Carpentries: https://carpentries.org/governance/

Projects for WG
- Glossary of Terms
- Proposed Guide for Accessibility - 
- Accessibility policy for Github (Accessibility.md)
-  Fireside Chat - in 2023?

Actions
- AAA: finding meeting time that works, and posting into the issue 
- ALS: finishing a rough accessibility.md (more definitions - less legal compliance!)... Will create working time for the next month to work on ACCESSIBILITY.md file (while we plan for recurring team meeting)
- LA: inviting Metadocencia team members
- AST: starting a new job!! 
- LH: Share your resources & people you know who can be consulted, will be available for consultation in coming weeks

**Comment by aleesteele** [comment]
## 04-07-2023
- Attendees: Liz (@LizHareDogs), Anne (@aleesteele)
- Reviewing introduction
- Nothing about us without us - can we add this? - https://en.wikipedia.org/wiki/Nothing_about_us_without_us
- How specific do we keep this document? Focused on TTW or expanded more broadly? (Bring up at next meeting!
- Core Principles: review next time
- Format of document: key themes seem to work well for now
- Will be updating existing document with introduction as it is now

**Comment by aleesteele** [comment]
## 10-07-2023
**Attendees**: Mariela Rajngewerc, Liz Hare (@LizHareDogs), Sophia Batchelor (@BrainonSilicon), Anne Lee Steele (@aleesteele)

- Mariela: Joining from Metadocencia, part of small team of folks from team working on accessibility, interested in translating materials that 
- Liz: When I got involved with the ManyDogs project, I found The Turing Way to be a valuable resource for open science-related things, thinks that TTW can be a resource for many things (& a starting point for many people). Gerat to
- Sophia: Working around HCI & how people use technology, excited to learn from this group, and make things accessible from a design & development stand-up

How has TTW been working on accessibility-related topics so far?
- Policies: Developing an accessibility.md (accessibility policy), currently being drafted on [Github](https://github.com/alan-turing-institute/the-turing-way/pull/3146/files)
- Guidelines: Chapter draft around [accessibility and events](https://pad.sfconservancy.org/p/ttw-etherpad), [github issue](https://github.com/alan-turing-institute/the-turing-way/issues/3075)
- Chapters: [Proposal for new guide for Accessibility](https://github.com/alan-turing-institute/the-turing-way/issues/2730), [Alt text for data visualisations](https://github.com/alan-turing-institute/the-turing-way/issues/2630)

How do we want these meetings to be organised? How do we want these spaces to be used?
- SB: Can we split between working sessions & people working on a specific topic, 
- LH: Discussion and coworking can be a lot (and might be a longer meeting)
- ALS: Having break-out rooms & meeting time? Can we change the meeting time?

Meeting logistics (deciding together):
- 2nd Monday of every month: verbal meeting & discussion
- 4th Monday of every month: coworking time? (in the future)
- Currently: coworking time on accessibility.md file!

What does Metadocencia do around accessibility right now?
- MR: Is the learning material itself accessible? Making some blogs about how to make an event, and to start making accessible slides. Example: https://www.metadocencia.org/en/post/accessibility_profiles/
- LH: do you have a process for reviewing accessibility? checklist? or only based on feedback?
- ALS: SB asked a question around enforcement and accountability
- LH: Related to CoC, for example getting conference attendees to agree to accessibility policy when they submit abstract for talk
- SB: How would we address the case where people might develop a resource or topic for TTW that might now be accessible by default, what do we do? Only researchers attend our AIM-RSF talks now that used to be general-audience? Who does the labor of accessibility?
- LH: In the TTW, there would definitely want to be able have that written work in support.
- ALS: I think we're asking two different questions around accessibility: How do we define accessibility in the first place? Two is how do we operationalise accessibility with the TTW community more broadly and supporting this kind of infrastructure needed to be able to make this work happen more broadly?
- LH: Can we encourage authors to write their own alt-text? We [can't] require it, but we can create an environment where people feel comfortable? 
- ALS: Do we police this kind of behaviour? 
- LH: We should! Who has the the responsibility of doing this work? The TTW is so collaborative, so there should be 
- ALS: What's the relationship with formalisation MR & SB? 
- LH: Formalisation is good! It holds people accountable?
- MR: We have the Accessibility Profiles that we are developing, but we also don't have a formalisation process so far
- SB: ALS can you rephrase?
- ALS: In thinking through Accessibility, what should be formalised and what should not?
- SB: I work with [AutSPACES](https://github.com/alan-turing-institute/AutSPACEs), and we were working around cognitive load in making materials accessible... we worked with colorblindedness & padding, and went through web accessibility guidelines? Can we think about pages being much more widely accessible?

What are we going to do next?
- MR: Bringing more background information, links, foci from Metadocencia (with reading on Accessibility policy)
- SB: Bringing more background information, links, foci from AutSPACES
- LH: Outlining text about alt-text, but I think the policy is more important, focused on both alt-text for data viz as well as accessibility policy
- ALS: All are invited to coworking on accessibility policy; LH + ALS will present more concentrated work at next meeting

**Reply by BrainonSilicon** [reply]
This is awesome @aleesteele

**Reply by aleesteele** [reply]
Link list from today:
- Accessibility talk: https://www.youtube.com/watch?v=ARMs7C_wE04&t=344s
- Accessibility for events draft: https://pad.sfconservancy.org/p/ttw-etherpad
- Accessibility guide proposal: https://github.com/alan-turing-institute/the-turing-way/issues/2730
- Accessibility policy for community: https://github.com/alan-turing-institute/the-turing-way/pull/3146/files
- Accessibility profiles from Metadocencia: https://www.metadocencia.org/en/post/accessibility_profiles/
- AutSPACES: https://github.com/alan-turing-institute/AutSPACEs
- Slack policies (have elements of accessibility): https://hackmd.io/@turingway/ttw-slack-intro

**Reply by trallard** [reply]
Sorry I missed this. I will have a couple of conflicts as I have training sessions now on Monday. 

Since there seems to be a lot of discussion points around accountability and accessibility in collective resources I want to share our accessibility checklist for content based PRs at Quansight. https://github.com/Quansight/Quansight-website/pull/748

We have flavours of this checklist in some of our open source projects (see https://github.com/nebari-dev/nebari-docs/pull/335) and also internally for events, meeting notes, slides (we have created templates with good accessibility affordances so folks can use for presentations), data visualization (we have a design system that is being made into a Python package).

A lot of this accountability works two ways - we train people, collate resources and work on tooling and materials with already good accessibility affordances so folks have a good starting point. 
Having the accessibility checklist within our PRs has also helped a lot with education and to signal that we as an organisation or community have standards and expectations around accessibility. And that we are all responsible for observing such standards, because for us creating accessible resources is as important as creating technically accurate or interesting content. 
Another example, pdfs such as our annual report not only go through writing, design, and editorial work, but we do accessibility checks during all these phases as well as at signing off and everyone is responsible for the accessibility within their area and function. 

It has taken us a while to get here, our journey has been progressive and iterative, and a lot of this has been as I said educational or grassroots advocacy, but it is worth it. 
We now have accessibility docs, policies, and processes that we observe on everything we do and we keep expanding but the single greatest game changer has been constantly reminding folks that everyone is responsible for the accessibility of the resources we create not a couple of folks, everyone.

**Comment by aleesteele** [comment]
## 11-07-2023
Attendees: Liz (@LizHareDogs), Anne (@aleesteele)

- Inspiration porn & poverty porn
- Nothing about us without us - can we add this? - https://en.wikipedia.org/wiki/Nothing_about_us_without_us
- How do we upstream contributions, suggestions & reflections about accessibility upstream to JupyterBook? i.e. https://github.com/jupyter/accessibility
- Important to add in process for implementation & importance
- JupyterBook Accessibility group: https://jupyter-accessibility.readthedocs.io/en/latest/
- Accessibility meetings with the Jupyter team: https://hackmd.io/WnaWXboXSiGoqWvev_fAvA

**Comment by LizHareDogs** [comment]
Here's the link to Stella Young's talk where she introduces "inspiration porn"

https://www.ted.com/talks/stella_young_i_m_not_your_inspiration_thank_you_very_much

**Comment by aleesteele** [comment]
## 4 July
- Attendees: Liz, Anne
- Nothing about us without us - can we add this? - https://en.wikipedia.org/wiki/Nothing_about_us_without_us
- How specific do we keep this document? Focused on TTW or expanded more broadly? (Bring up at next meeting!
- Core Principles
- Make sure to research 'nada sobre nosotros, sin nosotros'

**Comment by aleesteele** [comment]
## 4 August

- Attendees: Mariela, Liz, Anne
- Update: move to Github Organisation!!
- https://github.com/the-turing-way/the-turing-way
- https://github.com/TWTranslation
- https://www.youtube.com/watch?v=YoJe8Q4jaGg
-  today we'll be focused more on meta-practices - i.e. how to organise ourselves, given our new github organisation – and think about suggestions for accessibility for the infrastructure working group as they think through the repository (and how it might be split off into more spaces!). 
- ALS: How did you interact with the TTW repository?
- MR: It was clear for me. Read a bit, then came back.
- ALS: Was it clear how we use Github?
- MR: Always went to the links that were very orientated. I don't know if someone arrives to the Github. Went to look for specific issues, and I worked previously with Github. I kknew how to search, but might not be the case with all my colleagues 
- ALS: Are there Github repositories you look towards? Have you contributed to Github in the past?
- MR: We use Github at MetaDocencia. use the command line basically, sometimes merge. That's my experience, the people in Metadocencia. We use it very basically, we upload or change or edit. Sometime s some people come from computer science, so they know a bit more. Ivan comes from law - he doesn't interact with Github. For him, it's difficult in that he doesnt' speak Github. He doesn't speak English, so the comments are everything and its so new for him. That's more complicated. We don't use it !
- MR: When we work with him, we use Google Drive.
- ALS: Hugely important question, as it ties to how we engage with disciplines, i.e. people who work on ethics or law but don't have experience in . I love using the command line (i feel like a hacker!), but I know it's not accessible for everyone.
- LH: The transition is great. The possibility of organising in different repositories will be potentally cognitively easier, but the idea of building this accessibility chapter and how to add the content to it. There are so many levels, and it's kind of confusing, and I have to reorient myself each time. Having multiple repositories for each guide could be good - i.e. having a separate repo for the accessibility guide, until we're ready to bring it back. Maybe translating the training in different languages.
- LH: Enhanced transcript for screen-readers for videos that have descriptions of what to do - step-by-step guides on how to use Github. 
- MR: My colleague would also benefit from having this
- LH: When I apply for things, The Turing Way github organisation sticker helps.
- Github policies - Accessibility policies
- https://pad.sfconservancy.org/p/ttw-etherpad
- MR: started to do research around accessibility policies. Have a proposal policy to think about the policy. A group of people are going to be working in OLS to work on governance. They are going to go deeper in this talking about their policies. 
- MR: They are general ideas, but realised how to think through the details of how Metadocencia is thinking about its policies.
- ALS: what was the process?
- MR: Sharing two documents (not listed here)
- ALS: Can we help with this process? Translation or otherwise? Do you want to upload them anywhere in the TTW?
- LH: Important to recognise that document's are Metadocencía's.
- MR: What do you think so? 
- LH: The importance of recognition. 
- ALS: Could be colonial implications?
- MR: Would have to speak to my team. Good point.
- LH: If there's anything we could do to give you credit or support, I hope you would be welcome to tell us. 
- ALS: We build the world we want!

Planning for next meeting:
- Attendees: Liz, Mariela, Laurel, Tania (to be confirmed)
- Next things
- Updates for our accessibility policy
- Reviewing the documents that MR shared
- Async discussions on how we organise between ourselves
- ALS: archiving notes so far in an accsesibility WG repository for
-  LH: Will definitely help with cognitive load.

**Comment by aleesteele** [comment]
## 17 August

- ALS: Reflecting on remote collaboration & introversion and this slack post today. 
- LH: I actually really like collaboration in real time: 
- Open and equitable seed awards: https://www.openandequitable.org/seed-awards
- Access computing: https://www.washington.edu/accesscomputing/apply-accesscomputing-minigrant
- ALS: What are our reflections on other policies?
- LH: There are lots of policies that focus on aims, but there. A lot of people who have disabilities might be able to tell you what's wrong with your website, but they might not have the resources or training to do something about it! 
- ALS: Yes, there's a lot of narrative policy in setting the policy that gives us the language to engage with with access-related questions
- LH: Yes, they set an expectation of how to engage.
- AAA: Creating an exception for enabling Liz to apply for funding. Funder wants to understand what communities will benefit from you being a part of the community and with the Book Dash. Can you write a paragraph of what you bring to the table Liz about applying for the grant?
- LH: When you're talking about validation or colleagues of 10-15 people?
- AAA: The group wants to make sure they can keep track of the funding to different people with different backgrounds, so we'll have to give them some evidence beforehand and in advance.
- LH: I have people that I could recommend, but will have to think on it. 
- AAA: Will be good for networking and resources for you as well. This funding can start with TTW, but can be used for other projects in the future.
- ALS: What will the book dash committee be like with repect to this?
- AAA: Liz will set the standard. Two hours per week from beginning of September. Will send a 
- LH: Can the funding be applied for the accessibility policy as well?
- ALS: Is the deadline for the accessibility policy for Book Dash?
- AAA: Funded activities should take place before August 1, 2024.
- ALS: Timeline for funding?
- AAA: By Monday, I will send spreadsheet of estimated duties for Book Dash. LH will send information
- ALS: Do we want to make github repositories for each working group or a single folder with all information? Maybe this is a question for the core team meeting? 
- AAA: Good question. Maybe this should be something that is asked in the Core Team Meeting in the autumn? Important that we phrase what is happening here for other working groups to be able to support other people. This funding can be important for fielding that support for other people and sharing that information. We can support applications 
- ALS: I will add LH to core team channels, calendar invites, and otherwise. 
- ALS: Reviewing the policy - asking people questions.
- AAA: More practical in the community handbook, and more about the 'why' in the policy. If I want to do a book dash, I want to go to the handbook. But if I want to learn about the why, then I would check the policy.
- LH: Hadn't thought about that! This relationship between the community handbook and the policy
- ALS: I will bring a framework for how we divide it out to next meeting.

**Comment by aleesteele** [comment]
All notes have been added to #2 in the accessibility working group repository
